src/debug_menu.cpp

```diff
@@ -3903,15 +3903,7 @@ void debug()
     const bool should_disable_achievements = action && !is_debug_character() &&
             !non_cheaty_options.count( *action );
     if( should_disable_achievements && achievements.is_enabled() ) {
-        static const std::string query(
-            translate_marker(
-                "Using this will disable achievements.  Proceed?"
-                "\nThey can be reenabled in the 'game' section of the menu." ) );
-        if( query_yn( _( query ) ) ) {
-            achievements.set_enabled( false );
-        } else {
-            action = std::nullopt;
-        }
+        achievements.set_enabled( false );
```

Comment:
I'd prefer if it remind it's turned off achievements in hopes that I spend less time when testing achievements with them turned off.

Reply:
Does the notice on the achievements page not work for this?

I've never had to test achievements, but I presume you would be viewing the achievements page.

Reply:
Readded as a log message - 876c53a071238d2dad90d15199f1260bfea59443